model/Core/Vocabularies/RelationshipType.md

```diff
@@ -43,11 +44,11 @@ name completes the sentence:
 - fixedIn: (Security/VEX) A `from` Vulnerability has been fixed in each of the `to` Element(s). The use of the `fixedIn` type is contrained to `VexFixedVulnAssessmentRelationship` classed relationships.
 - foundBy: (Security) Designates a `from` Vulnerability was originally discovered by the `to` Agent(s)
 - generates: The `from` Element generates each `to` Element
-- hasAddedFile: Every `to` Element is is a file added to the `from` Element (`from` hasAddedFile `to`)
+- hasAddedFile: Every `to` Element is a file added to the `from` Element (`from` hasAddedFile `to`)
 - hasAssessmentFor: (Security) Relates a `from` Vulnerability and each `to` Element(s) with a security assessment. To be used with `VulnAssessmentRelationship` types
 - hasAssociatedVulnerability: (Security) Used to associate a `from` Artifact with each `to` Vulnerability
 - hasConcludedLicense: The `from` Software Artifact is concluded by the SPDX data creator to be governed by each `to` license
-- hasDataFile: The `from` Element treats each `to` Element as a data file
+- hasDataFile: The `from` Element treats each `to` Element as a data file. A data file is a data resource that the `from` Element can use to fulfil its expected functinality or behavior. It can be a generic database fie, a log file, an AI model file, a calibration data file, a temporary file, etc. For AI training data, test data, other test artifacts, configuration data, Build input data, and Build output data, please consider using `trainedOn`, `testedOn`, `hasTest`, `configures`, `hasInputs`, and `hasOutputs` types, respectively.
```

Comment:
1. Generic database fie --> File 
2. I am unsure if saying hasDataFile is the right name? Should it hasAsset? because hasDataFile for a model executable and log file seems odd and non-intitutive. 
3. Also, I am unsure and maybe we should clarify how its different from dependsOn relationship?

Reply:
On (2), we cannot change the name in 3.0.x, it is already published. But it is possible in 3.1.

(3) It will be helpful to have that. One thing I can see is that `hasDataFile` doesn't imply dependency. Without the `to` Element, the `from` Element can still functional.

Reply:
Okay. I am okay with it and maybe we can note down somewhere it should be hasArtificat as the name. But otherwise, I approve